src/analyses/reaching_definitions.h

```diff
@@ -113,6 +148,18 @@ class rd_range_domaint:public ai_domain_baset
     bv_container=&_bv_container;
   }
 
+  /// Computes an instance obtained from the instance `*this` by transformation
+  /// over a GOTO instruction referenced by `from`. The method implements a
+  /// switch according to a type of the instruction and then calls a dedicated
+  /// transform_*` method for the recognised instruction.
```

Comment:
Doxygen is getting confused because of a missing backtick at the beginning of line 154.  Adding the backtick should fix the warnings.  (@marek-trtik It's easy to see this problem by looking at the documentation produced by Doxygen.  I suggest reviewing what the generated documentation looks like in `doc/html`.)

```suggestion
  /// `transform_*` method for the recognised instruction.
```

Reply:
Thanks! Fixed.